src/spikeinterface/core/waveform_extractor.py

```diff
@@ -1990,6 +1994,9 @@ def _load_extension_data(self):
                     ext_data = pd.read_csv(ext_data_file, index_col=0)
                 elif ext_data_file.suffix == ".pkl":
                     ext_data = pickle.load(ext_data_file.open("rb"))
+                else:
+                    print(f"{ext_data_file.name} not created by spikeinterface, skipping this file")
+                    continue
```

Comment:
Not sure to see the use case.
Why would there be a file not created by spikeinterface is this subfolder ?

Reply:
oups sorry. I haven't read in detail the comments.